cpp/perspective/src/cpp/context_grouped_pkey.cpp

```diff
@@ -42,9 +42,25 @@ t_ctx_grouped_pkey::init() {
     m_tree = std::make_shared<t_stree>(pivots, m_config.get_aggregates(), m_schema, m_config);
     m_tree->init();
     m_traversal = std::shared_ptr<t_traversal>(new t_traversal(m_tree));
+

```

Comment:
Unrelated to this PR but - we should not compile this as there is no public API (nor tests ...)